hack/generate-seed-crds.sh

```diff
@@ -34,6 +34,7 @@ fi
 output_dir="$(pwd)"
 file_name_prefix="$1"
 
+# TODO generate HVPA CRD when https://github.com/gardener/hvpa-controller/issues/90 is resolved
```

Comment:
You could consider the following diff. We should try to assign an owner for each TODO we introduce to make sure that we follow-up on them.

```diff
diff --git a/hack/generate-seed-crds.sh b/hack/generate-seed-crds.sh
index 2fb85bdea..55d061044 100755
--- a/hack/generate-seed-crds.sh
+++ b/hack/generate-seed-crds.sh
@@ -85,6 +85,9 @@ if [ -n "${2:-}" ]; then
     shift
   done
 else
+  # TODO(kris94): Currently it is not possible to generate the hvpas.autoscaling.k8s.io CRD with controller-gen due to https://github.com/gardener/hvpa-controller/issues/90.
+  # Generate the hvpas.autoscaling.k8s.io CRD once the corresponding issue is resolved.
+
   generate_group extensions.gardener.cloud
   generate_group resources.gardener.cloud
   generate_group druid.gardener.cloud
```

Reply:
Ref https://github.com/gardener/hvpa-controller/issues/90#issuecomment-1079031790. We should first update to 
 `hvpa-controller@v0.4.0`, then controller-gen should be able to generate the hvpa CRDs.